resources/deployments/model/deployments.go

```diff
@@ -34,6 +34,7 @@ type DeploymentsModel struct {
 	deviceDeploymentLogsStorage DeviceDeploymentLogsStorage
 	imageLinker                 GetRequester
 	deviceDeploymentGenerator   Generator
+	imageContentType            string
 }
 
 func NewDeploymentModel(
```

Comment:
makes sense, doe how we would handle incorrectly constructed config? return extra error from New ?